finetune/target_models/masked_language_model.py

```diff
@@ -29,9 +29,9 @@ def feed_shape_type_def(self):
                 {
                     "tokens": TS([None, 2]),
                     "mask": TS([None]),
-                    "mlm_weights": TS([self.config.max_masked_tokens * self.config.batch_size]),
-                    "mlm_ids": TS([self.config.max_masked_tokens * self.config.batch_size]),
-                    "mlm_positions": TS([self.config.max_masked_tokens * self.config.batch_size]),
+                    "mlm_weights": TS([None]),
```

Comment:
sweet. one thing to note is that now the len dim could be variable, which means that the return type if predicting in a batch might be an iterable instead of an array. though it aren't really doing any batch prediction anyway so it probably doesn't matter.

Reply:
I'm not sure I follow.

Reply:
@lhz1029 that's not the case. this is flattened in all instances

Reply:
it's not the case that the seq len dim could be variable?

Reply:
Oh whoops I was thinking about an imaginary `predict` method in MLM that does not currently exist